src/reason-parser/reason_pprint_ast.ml

```diff
@@ -2570,6 +2570,22 @@ let formatComputedInfixChain infixChainList =
   print [] [] "" infixChainList
 
 
+let rec injectClosingRParen = function
```

Comment:
I think if we match Prettier's style, nothing special would be needed for injecting the closing paren, right?